src/routes/index.tsx

```diff
@@ -5,20 +5,25 @@ import DailyDropffsReport from "../components/daily-dropoffs-report";
 import ProofingOverTimeReport from "../components/proofing-over-time-report";
 import { Router } from "../router";
 import HomeRoute from "./home-route";
-import createReportRoute from "./report-route";
+import createReportRoute, { ReportRoute } from "./report-route";
 import { Scale } from "../contexts/report-filter-context";
+import ALL_ROUTES from "./all";
 
-export const ROUTES = {
+/**
+ * Requires that all keys in ALL_ROUTES have a matching key in this object
+ */
+type ReportRoutes = {
+  [Property in keyof typeof ALL_ROUTES]: ReportRoute;
+}
+
+const reportRoutes: ReportRoutes = {
```

Comment:
For future consideration, we may be able to collapse the route constant back in this file if we moved out the static imports which were causing it trouble, using [`React.lazy`](https://reactjs.org/docs/code-splitting.html).

Thinking something like...

```ts
const ROUTES = {
  '/daily-auths-report': {
    title: 'Daily Auths Report',
    component: lazy(() => import('./daily-dropoffs-report')),
  },
};
```

Reply:
I think we'd have to use [preact-lazy](https://www.npmjs.com/package/preact-lazy) right?

Reply:
Oh, right, I forgot we're using Preact here! I'm not super familiar with that specific library, but I imagine it's probably going for the same idea, yeah.

Reply:
Though it looks like `preact/compat` has a `lazy` function:

https://preactjs.com/guide/v10/switching-to-preact#suspense

Reply:
sounds like a problem for future us 😎